tests/src/test/java/alluxio/client/fuse/dora/FuseFileSystemDataTest.java

```diff
@@ -387,17 +387,17 @@ public void getAttrWhenWriting() {
         DEFAULT_MODE.toShort(), mFileInfo.get()));
     ByteBuffer buffer = BufferUtils.getIncreasingByteBuffer(DEFAULT_FILE_LEN);
     Assert.assertEquals(DEFAULT_FILE_LEN,
-        mFuseFs.write(path, buffer, DEFAULT_FILE_LEN, 0, mFileInfo.get()));
+        mFuseFs.write(path, 0, buffer, mFileInfo.get()));
     Assert.assertEquals(0,
         mFuseFs.getattr(path, mFileStat));
     Assert.assertEquals(DEFAULT_FILE_LEN, mFileStat.st_size.intValue());
     Assert.assertEquals(AlluxioFuseUtils.getSystemUid(), mFileStat.st_uid.get());
     Assert.assertEquals(AlluxioFuseUtils.getSystemGid(), mFileStat.st_gid.get());
     Assert.assertEquals(AlluxioFuseUtils.getSystemGid(), mFileStat.st_gid.get());
     Assert.assertTrue((mFileStat.st_mode.intValue() & FileStat.S_IFREG) != 0);
-    buffer.flip();
+    buffer.clear().limit(DEFAULT_FILE_LEN);
     Assert.assertEquals(DEFAULT_FILE_LEN,
-        mFuseFs.write(path, buffer, DEFAULT_FILE_LEN, 0, mFileInfo.get()));
+        mFuseFs.write(path, DEFAULT_FILE_LEN, buffer, mFileInfo.get()));
     Assert.assertEquals(0,
         mFuseFs.getattr(path, mFileStat));
     Assert.assertEquals(DEFAULT_FILE_LEN * 2, mFileStat.st_size.intValue());
```

Comment:
The position is set to `DEFAULT_FILE_LEN` instead of the original `0` because otherwise it's overwriting the data it has just written from the beginning, and FUSE will silently ignore such writes. This will contradict the expectation below that the file size will be 2x `DEFAULT_FILE_LEN`.